lib/hanami/commands/generate/app.rb

```diff
@@ -7,11 +7,14 @@ module Commands
     class Generate
       class App < Abstract
 
+        DEFAULT_TEMPLATE_FORMATS = 'erb'.freeze
+
+        APPLICATION_TEMPLATE_FORMATS = %w[erb slim haml].freeze
```

Comment:
Do you have any ideas how I can create any template dynamically? For example, I have `template.erb`:
``` erb
<div><%= text %></div>
```

And I need to get template.slim like this:
``` slim
div
  = text
```

Or `template.haml`:
``` haml
%div
  = text
```